utils/default-helpers/src/main/scala/pl/touk/nussknacker/engine/util/functions/collection.scala

```diff
@@ -0,0 +1,135 @@
+package pl.touk.nussknacker.engine.util.functions
+
+import pl.touk.nussknacker.engine.api.generics.GenericType
+import pl.touk.nussknacker.engine.api.{Documentation, HideToString, ParamName}
+
+import java.util.{Collections, Objects}
+import scala.collection.immutable.ListMap
+import scala.jdk.CollectionConverters.{CollectionHasAsScala, _}
+
+object collection extends HideToString {
+
+  @Documentation(description = "Concatenates two lists")
+  @GenericType(typingFunction = classOf[TypingFunctions.ListAdditionTyping])
+  def concat[T](@ParamName("list1") list1: java.util.List[T], @ParamName("list2") list2: java.util.List[T]): java.util.List[T] = (list1.asScala.toList ++ list2.asScala).asJava
```

Comment:
I know that we are writing in Scala, but I believe that library-level routines should be efficient - how about pre-allocating an ArrayList of proper size and adding both lists into that?

Reply:
(also - we don't enforce any coding style, but this line is 176 characters long, can you wrap it? or buy me 4k screen? 😉)